config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -258,6 +258,46 @@ presubmits:
       testgrid-tab-name: pr-cloud-provider-azure-e2e-ccm-vmss
       description: "Runs Azure specific tests with cloud-provider-azure (https://github.com/kubernetes-sigs/cloud-provider-azure)."
       testgrid-num-columns-recent: '30'
+  - name: pull-cloud-provider-azure-e2e-capz
```

Comment:
`pull-cloud-provider-azure-e2e` enables `MixedProtocolLBService` feature gate. Have you tested if the capz cluster enabled it as well?

Reply:
I checked a cluster generated with capz and the feature gate is not enabled. I will update the code.

Reply:
I added env var `TEST_CCM: "true"` for `pull-cloud-provider-azure-e2e-capz`, then this check has the same yaml file as the already existing `pull-cloud-provider-azure-e2e-ccm-capz`. Besides, according to the old aks-engine checks' config files, these 2 checks should have the same cluster config.